### DISCUSSION TITLE
Bug Report Guide

### DISCUSSION THREAD

**Discussion by aaronliu0130** [upscayl/upscayl#392] [Q&A] [answered]
1. 🧐 Make sure you're filing an actual bug report about the software and read [the FAQ](https://github.com/upscayl/upscayl/tree/main#-faq)
2. 🛫 Go to [Upscayl's issue forms](https://github.com/upscayl/upscayl/issues/new/choose)
3. ✔️ Follow the checklist at the top of the form
4. 📋 If applicable, send the logs by doing [this](https://github.com/upscayl/upscayl/wiki/Guide#logs)
5. 🧑‍💻 Fill out your GPU and system version (unless u use arch)
6. ✍️ Fill out all other applicable fields and submit!

**Comment by bluedolphin629** [comment]
Problem when I just started using it: vertical strips. Out of service。and i don't know how tu do.please help me !
<img width="1250" alt="微信图片_20231125025635" src="https://github.com/upscayl/upscayl/assets/151951281/daa8c539-b191-49e2-8685-2bb626cb14d6">

**Reply by aaronliu0130** [reply]
Please follow the guide.

**Comment by champatgithub** [comment]
First of all, Thank you for this great software guys, Recently I ran into an issue. Why is this happening, I've tried with every setting possible. Is this a bug or something else?

![a-young-tattooed-27-year-old-latino-woman-with-lon (1)_upscayl_2x_ultramix_balanced](https://github.com/upscayl/upscayl/assets/171545105/456a12b7-1548-4302-b3a1-6d084dbd2e83)


![Screenshot 2024-06-03 221157](https://github.com/upscayl/upscayl/assets/171545105/a8d37ed7-5e70-4be8-89a3-a725815929cb)

**Reply by aaronliu0130** [reply]
Please update. Next time, follow the guide.

**Reply by champatgithub** [reply]
Sr. what do you mean by update?

**Reply by NayamAmarshe** [reply]
You're probably using compression for PNG. Try turning it off

**Reply by aaronliu0130** [reply, marked as answer]
Update to 2.11.5. If there isn't an auto-updater, re-download the latest version of upscayl.

**Reply by champatgithub** [reply]
> You're probably using compression for PNG. Try turning it off

No Sr. Did not use the compression

**Reply by champatgithub** [reply]
> Update to 2.11.5. If there isn't an auto-updater, re-download the latest version of upscayl.

Finally Fixed it by resetting the Upscayl 3 to 4 times

**Reply by champatgithub** [reply]
Thank you @aaronliu0130 and @NayamAmarshe for taking your valuable time to help 🙏

**Comment by Elysia-yyy** [comment]
💻 System Info: {"platform":"win","release":"10.0.22631","arch":"x64","model":"12th Gen Intel(R) Core(TM) i7-12650H","cpuCount":16,"gpu":{"active":true,"cudaComputeCapabilityMajor":0,"deviceId":65535,"deviceString":"ANGLE (Google, Vulkan 1.3.0 (SwiftShader Device (Subzero) (0x0000C0DE)), SwiftShader driver-5.0.0)","driverVendor":"SwANGLE","driverVersion":"5.0.0","gpuPreference":0,"revision":0,"subSysId":0,"vendorId":65535,"vendorString":"Google Inc. (Google)"}}
🔄 Resetting image paths
🎒 BACKEND REPORTED:  🖼️ Updating Image Path:  D:\图\镜流\NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!.png
🎒 BACKEND REPORTED:  📄 Selected File Path:  D:\图\镜流\NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!.png
🖼 Selected Image Path:  D:\图\镜流\NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!.png
📁 Selected Image Directory:  D:\图\镜流
🖼 imagePath:  D:\图\镜流\NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!.png
🔤 Extension:  png
🔄 Resetting Upscaled Image Path
🏁 UPSCAYL
🎒 BACKEND REPORTED:  ✅ Upscayl Variables:  {"model":"upscayl-standard-4x","gpuId":null,"saveImageAs":"png","inputDir":"D:\\图\\镜流","fileNameWithExt":"NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!.png","outputDir":"D:\\图\\镜流","outFile":"D:\\图\\镜流\\NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!_upscayl_4x_upscayl-standard-4x.png","fileName":"NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!","scale":"4","compression":"0","customWidth":"","useCustomWidth":false,"tileSize":null}
🎒 BACKEND REPORTED:  📢 Upscayl Command:  -i,D:\图\镜流\NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!.png,-o,D:\图\镜流\NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!_upscayl_4x_upscayl-standard-4x.png,-m,D:\upscayl\resources\models,-n,upscayl-standard-4x,-f,png,-c,0
🎒 BACKEND REPORTED:  👶 Updating Child Processes:  {"binary":"D:\\upscayl\\resources\\bin\\upscayl-bin","args":["D:\\upscayl\\resources\\bin\\upscayl-bin","-i","D:\\图\\镜流\\NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!.png","-o","D:\\图\\镜流\\NR8AVjViQ2dBeE1EQTVNVEk0TnpNMnBSQ0phVlR4TlJjIQUAcXVuZ3o!_upscayl_4x_upscayl-standard-4x.png","-m","D:\\upscayl\\resources\\models","-n","upscayl-standard-4x","-f","png","-c","0"]}
🎒 BACKEND REPORTED:  🛑 Updating Stopped:  false
🎒 BACKEND REPORTED:  [0 NVIDIA GeForce RTX 4060 Laptop GPU]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[0 NVIDIA GeForce RTX 4060 Laptop GPU]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce RTX 4060 Laptop GPU]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 NVIDIA GeForce RTX 4060 Laptop GPU]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1
[1 Intel(R) UHD Graphics]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[1 Intel(R) UHD Graphics]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 Intel(R) UHD Graphics]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[1 Intel(R) UHD Graphics]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [0 NVIDIA GeForce RTX 4060 Laptop GPU]  queueC=2[8]  queueG=0[16]  queueT=1[2]
[0 NVIDIA GeForce RTX 4060 Laptop GPU]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce RTX 4060 Laptop GPU]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 NVIDIA GeForce RTX 4060 Laptop GPU]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1
[1 Intel(R) UHD Graphics]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[1 Intel(R) UHD Graphics]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 Intel(R) UHD Graphics]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[1 Intel(R) UHD Graphics]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🎒 BACKEND REPORTED:  0.00%

🚧 UPSCAYL_PROGRESS:  0.00%

🎒 BACKEND REPORTED:  1.85%

🚧 UPSCAYL_PROGRESS:  1.85%

🎒 BACKEND REPORTED:  3.70%

🚧 UPSCAYL_PROGRESS:  3.70%

🎒 BACKEND REPORTED:  5.56%

🚧 UPSCAYL_PROGRESS:  5.56%

🎒 BACKEND REPORTED:  7.41%

🚧 UPSCAYL_PROGRESS:  7.41%

🎒 BACKEND REPORTED:  vkWaitForFences failed -4

🚧 UPSCAYL_PROGRESS:  vkWaitForFences failed -4

🔄 Resetting image paths
🎒 BACKEND REPORTED:  9.26%
vkQueueSubmit failed -4

🚧 UPSCAYL_PROGRESS:  9.26%
vkQueueSubmit failed -4

🔄 Resetting image paths
显卡是rtx4060，系统是win11家庭版23h2
我这次总计提升了700多张图，就只有这一张图不管怎么改都在7.41的时候崩溃，所有图片大小都差不多
